BDArmory/Distribution/GameData/BDArmory/MMPatches/000000_HitpointModule_PartFixes.cfg

```diff
@@ -3841,6 +3841,13 @@
 ///////////////////////////////////////////
 //Fixes for BDA MISC
 ///////////////////////////////////////////
+@PART[seatExternalCmdweaponized] //combat seat
+{
+	%MODULE[HitpointTracker]
+	{
+		%masHitPoints = 500 //this way, this doesn't die beforethe kerbal in it does
```

Comment:
other entries in the MM patch aren't preceded by % (with the exception of `maxSupportedArmor`); what does the % do, or is it a typo? Also, I'm assuming you meant `maxHitPoints`, not `masHitPoints`.

Reply:
oops, yeah, it should be maxHitPoints. The % is an add-or-edit marker; since the MODULE is already getting modified via %, and nothing else is using it for node values, probably don't need ``%maxHitPoints`` specifically, could just be ``maxHitPoints``